docs/documentation.html

```diff
@@ -146,12 +146,15 @@
         <li><a href="#streams">9. Kafka Streams</a>
             <ul>
                 <li><a href="#streams_overview">9.1 Overview</a></li>
-                <li><a href="#streams_developer">9.2 Developer Guide</a></li>
+                <li><a href="#streams_concepts">9.2 Core Concepts</a></li>
+                <li><a href="#streams_architecture">9.3 Architecture</a></li>
+                <li><a href="#streams_developer">9.4 Developer Guide</a></li>
                 <ul>
-                    <li><a href="#streams_concepts">Core Concepts</a></li>
                     <li><a href="#streams_processor">Low-Level Processor API</a></li>
                     <li><a href="#streams_dsl">High-Level Streams DSL</a></li>
+                    <li><a href="#streams_execute">Application Configuration and Execution</a></li>
```

Comment:
2 lines above:

-> Low-level Processor API (PAPI)

We should try to establish this abbreviation. :)